versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1"
+  required_version = ">= 0.14"
```

Comment:
Why lower this pin ?

Reply:
Ran into this https://github.com/cloudposse/actions/runs/3653277654?check_suite_focus=true because `-get-plugins` was removed in TF 0.15. The test harness fails in `test/terraform/get-modules.bats`